Common/include/Gauss_Elimination_core.hpp

```diff
@@ -0,0 +1,141 @@
+    su2double d0,d1,d2,d3,d4;
+    if (nVar<6) {
+#define BLOCK(i,j) block[(i)*nVar+(j)]
+    d0 = 1.0000000000000000 / BLOCK(0,0);
```

Comment:
Why 1 is defined ass 1.00000000 . I´m sure there is a good reason for that. Thanks!